dace/memlet.py

```diff
@@ -293,3 +293,26 @@ class EmptyMemlet(Memlet):
 
     def __init__(self):
         super(EmptyMemlet, self).__init__(None, 0, None, 1)
+
+
+class MemletTree(object):
```

Comment:
This looks to be an interesting concept. Can you provide some description or examples of what you can do with it?

Reply:
Done.